components/system-api/src/private.rs

```diff
@@ -19,134 +19,101 @@
 //!
 //! # Table of Contents
 //!
-//! - [List peers](#list-peers)
-//! - [Add peer](#add-peer)
 //! - [Get node info](#get-node-info)
-//! - [Check if consensus is enabled](#check-if-consensus-is-enabled)
+//! - [Get node statistics](#get-node-statistics)
+//! - [Add peer](#add-peer)
 //! - [Change consensus status](#change-consensus-status)
 //! - [Node shutdown](#node-shutdown)
 //!
-//! # List Peers
+//! # Get Node Info
 //!
 //! | Property    | Value |
 //! |-------------|-------|
-//! | Path        | `/api/system/v1/peers` |
+//! | Path        | `/api/system/v1/info` |
 //! | Method      | GET   |
 //! | Query type  | - |
-//! | Return type | [`PeersInfo`] |
-//!
-//! Returns the list of incoming and outgoing connections for current node.
-//!
-//! [`PeersInfo`]: struct.PeersInfo.html
-//!
-//! ```
-//! use exonum_system_api::{private::PeersInfo, SystemApiPlugin};
-//! use exonum_testkit::{ApiKind, TestKitBuilder};
-//!
-//! # fn main() -> Result<(), failure::Error> {
-//! let mut testkit = TestKitBuilder::validator()
-//!     .with_plugin(SystemApiPlugin)
-//!     .build();
-//! let api = testkit.api();
-//! let info: PeersInfo = api.private(ApiKind::System).get("v1/peers")?;
-//! # Ok(())
-//! # }
-//! ```
-//!
-//! # Add Peer
-//!
-//! | Property    | Value |
-//! |-------------|-------|
-//! | Path        | `/api/system/v1/peers` |
-//! | Method      | POST   |
-//! | Query type  | [`ConnectInfo`] |
-//! | Return type | - |
+//! | Return type | [`NodeInfo`] |
 //!
-//! Adds a peer to the Exonum node. Node will attempt to connect to this peer.
+//! Obtains information about node.
 //!
-//! [`ConnectInfo`]: https://docs.rs/exonum-node/latest/exonum_node/struct.ConnectInfo.html
+//! [`NodeInfo`]: struct.NodeInfo.html
 //!
 //! ```
-//! use exonum_node::ConnectInfo;
-//! use exonum_system_api::SystemApiPlugin;
+//! use exonum_system_api::{private::NodeInfo, SystemApiPlugin};
 //! use exonum_testkit::{ApiKind, TestKitBuilder};
 //!
 //! # fn main() -> Result<(), failure::Error> {
-//! # let address = "127.0.0.1:8080".to_owned();
-//! # let public_key = Default::default();
-//! // Obtaining address and public key of target node skipped...
-//! let connect_info = ConnectInfo {
-//!     address,
-//!     public_key,
-//! };
-//!
 //! let mut testkit = TestKitBuilder::validator()
 //!     .with_plugin(SystemApiPlugin)
 //!     .build();
 //! let api = testkit.api();
-//! api.private(ApiKind::System)
-//!     .query(&connect_info)
-//!     .post("v1/peers")?;
+//! let info: NodeInfo = api.private(ApiKind::System).get("v1/info")?;
 //! # Ok(())
 //! # }
 //! ```
 //!
-//! # Get Node Info
+//! # Get Node Statistics
 //!
 //! | Property    | Value |
 //! |-------------|-------|
-//! | Path        | `/api/system/v1/network` |
+//! | Path        | `/api/system/v1/stats` |
 //! | Method      | GET   |
 //! | Query type  | - |
-//! | Return type | [`NodeInfo`] |
+//! | Return type | [`StatsInfo`] |
```

Comment:
Type name and link should be updated here to point to `NodeStats`. Weird that `deadlinks` didn't pick up this error. (`cargo clean --doc` should probably run before generating docs.)